### PR TITLE
Display image digest for container packages

### LINKED ISSUE #21160: Container registry show image hash
### Feature Description

Hello! I think it would be helpful to show the image hashes/digests for docker container images on the respective package page. I tend to reference them for verifying the build/push/pull process. I don't think Gitea currently has this, but if it does, I can't seem to find it. Thank you!

### Screenshots

Image

### PR DESCRIPTION
**Pull request by KN4CK3R**
fixes #21160

grafik

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: templates/package/content/container.tmpl [hunk 1]
```diff
@@ -14,6 +14,10 @@
 					<div class="markup"><pre class="code-block"><code>docker pull {{.RegistryHost}}/{{.PackageDescriptor.Owner.LowerName}}/{{.PackageDescriptor.Package.LowerName}}{{$separator}}{{.PackageDescriptor.Version.LowerVersion}}</code></pre></div>
 				{{end}}
 			</div>
+			<div class="field">
+				<label>{{svg "octicon-code"}} {{.locale.Tr "packages.container.digest"}}</label>
+				<div class="markup"><pre class="code-block"><code>{{range .PackageDescriptor.Files}}{{if eq .File.LowerName "manifest.json"}}{{.Properties.GetByName "container.digest"}}{{end}}{{end}}</code></pre></div>
```
Comment (delvh) [inline templates/package/content/container.tmpl:19]:
```suggestion
				<label for="container-image-digest">{{svg "octicon-code"}} {{.locale.Tr "packages.container.digest"}}</label>
				<div class="markup" id="container-image-digest"><pre class="code-block"><code>{{range .PackageDescriptor.Files}}{{if eq .File.LowerName "manifest.json"}}{{.Properties.GetByName "container.digest"}}{{end}}{{end}}</code></pre></div>
```
Should hopefully be more accessible while not changing its display.

**Review by delvh — APPROVED** [review]

File: templates/package/content/container.tmpl (hunk 1 above)
Comment (KN4CK3R) [reply templates/package/content/container.tmpl:19]: These should be added in another PR because none of the package labels use that.

**Comment by 6543** [comment]
:rocket:

File: templates/package/content/container.tmpl (hunk 1 above)
Comment (6543) [reply templates/package/content/container.tmpl:19]: looking forward to it :point_up:  ;)

**Pull request merged**